frontend/packages/dev-console/integration-tests/features/monitoring/monitoring-Page-Smoke-Metrics-ODC-4116.feature

```diff
@@ -0,0 +1,177 @@
+    And user will see the pods list inside the project
+    And user will see the value of Received Bandwidth by each pod
+
+ 
+@smoke
+Scenario: Transmit Bandwidth Query on Metrics tab: MH-01-TC04
+    Given user is at Developer perspective
+    When user opens Monitoring page
+    And user clicks on Metrics tab
+    And user clicks on Select Query dropdown
+    And user selects Transmit Bandwidth Query
+    Then user will see the Transmit Bandwidth for project
+    And user will see the pods list inside the project
+    And user will see the value of Trasmitted Bandwidth by each pod
+
+
+@smoke
+Scenario: Rate of Received Packets Query on Metrics tab: MH-01-TC04
+    Given user is at Developer perspective
+    When user opens Monitoring page
+    And user clicks on Metrics tab
+    And user clicks on Select Query dropdown
+    And user selects Received Packets Query
+    Then user will see the Received Packets for project
+    And user will see the pods list inside the project
+    And user will see the value of Received Packets by each pod
+
+ 
+@smoke
+Scenario: Rate of Trasmitted Packets Query on Metrics tab: MH-01-TC04
+    Given user is at Developer perspective
+    When user opens Monitoring page
+    And user clicks on Metrics tab
+    And user clicks on Select Query dropdown
+    And user selects Trasmitted Packets Query
+    Then user will see the Trasmitted Packets for project
+    And user will see the pods list inside the project
+    And user will see the value of Trasmitted Packets by each pod
+
+ 
+@smoke
+Scenario: Rate of Received Packets Dropped Query on Metrics tab: MH-01-TC04
+    Given user is at Developer perspective
+    When user opens Monitoring page
+    And user clicks on Metrics tab
+    And user clicks on Select Query dropdown
+    And user selects Received Packets Dropped Query
+    Then user will see the Received Packets Dropped for project
+    And user will see the pods list inside the project
+    And user will see the value of Received Packets Dropped by each pod
+
+ 
+@smoke
+Scenario: Rate of Trasmitted Packets Dropped Query on Metrics tab: MH-01-TC04
+    Given user is at Developer perspective
+    When user opens Monitoring page
+    And user clicks on Metrics tab
+    And user clicks on Select Query dropdown
+    And user selects Trasmitted Packets Dropped Query
+    Then user will see the Trasmitted Packets Dropped for project
+    And user will see the pods list inside the project
+    And user will see the value of Trasmitted Packets Dropped by each pod
+
+
+#Custom Query: "sum(container_memory_working_set_bytes{ namespace="openshift-monitoring", container!="POD", container!=""}) by (pod)"
+@smoke
+Scenario: Custom Query on Metrics tab: MH-01-TC04
+    Given user is at Developer perspective
+    When user opens Monitoring page
+    And user clicks on Metrics tab
+    And user clicks on Select Query dropdown
+    And user selects Custom Query
+    And user enters the custom query
```

Comment:
The query given above has capitals

Reply:
It's not a problem. It runs successfully.

Reply:
I'm talking about mentioning the variables correctly